### DISCUSSION TITLE
tft.setTextSize seems not working with decimal numbers

### DISCUSSION THREAD

**Discussion by fouderix** [Bodmer/TFT_eSPI#2151] [Q&A - Fonts]
Hello, I am trying to draw a text which is pretty small.
So I tried to do 
```
tft.setTextColor(TFT_WHITE, TFT_GREEN);
tft.setTextSize(0.5);
tft.setTextDatum(3);
tft.drawString(WiFi.SSID(ssid), 40, y, 4);
```
But the tft.setTextSize don't change anything.
Is it normal or it is a bug ?
Thanks

**Comment by Bodmer** [comment]
The library uses the Adafruit_GFX API for text size. The text multiplier must be an integer 1 to N. So text cannot be shrunk.

There is a GFX free font included in the library called TomThumb which is tiny.
